[2024-04-20T14:15:21.901+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: currency_parsing_pound.get_data manual__2024-04-20T14:01:23.073825+00:00 [queued]>
[2024-04-20T14:15:21.916+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: currency_parsing_pound.get_data manual__2024-04-20T14:01:23.073825+00:00 [queued]>
[2024-04-20T14:15:21.917+0000] {taskinstance.py:2193} INFO - Starting attempt 7 of 8
[2024-04-20T14:15:21.939+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): get_data> on 2024-04-20 14:01:23.073825+00:00
[2024-04-20T14:15:21.946+0000] {standard_task_runner.py:60} INFO - Started process 1280 to run task
[2024-04-20T14:15:21.952+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'currency_parsing_pound', 'get_data', 'manual__2024-04-20T14:01:23.073825+00:00', '--job-id', '876', '--raw', '--subdir', 'DAGS_FOLDER/stage/currency_pound.py', '--cfg-path', '/tmp/tmpaa6m8eq4']
[2024-04-20T14:15:21.954+0000] {standard_task_runner.py:88} INFO - Job 876: Subtask get_data
[2024-04-20T14:15:22.034+0000] {task_command.py:423} INFO - Running <TaskInstance: currency_parsing_pound.get_data manual__2024-04-20T14:01:23.073825+00:00 [running]> on host 31aa6d9ea6ff
[2024-04-20T14:15:22.123+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/context.py:207: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2024-04-20T14:15:22.178+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='currency_parsing_pound' AIRFLOW_CTX_TASK_ID='get_data' AIRFLOW_CTX_EXECUTION_DATE='2024-04-20T14:01:23.073825+00:00' AIRFLOW_CTX_TRY_NUMBER='7' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-20T14:01:23.073825+00:00'
[2024-04-20T14:15:22.180+0000] {taskinstance.py:2728} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 200, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 217, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/stage/currency_pound.py", line 10, in get_data
    prev_date = execution_date - timedelta(days=1)
TypeError: unsupported operand type(s) for -: 'str' and 'datetime.timedelta'
[2024-04-20T14:15:22.200+0000] {taskinstance.py:1149} INFO - Marking task as UP_FOR_RETRY. dag_id=currency_parsing_pound, task_id=get_data, execution_date=20240420T140123, start_date=20240420T141521, end_date=20240420T141522
[2024-04-20T14:15:22.221+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 876 for task get_data (unsupported operand type(s) for -: 'str' and 'datetime.timedelta'; 1280)
[2024-04-20T14:15:22.246+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-04-20T14:15:22.285+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
